schemas/system/Identifier.schema

```diff
@@ -0,0 +1,7 @@
+System({
+  Family: Enum('Registry', 'Dictionary', 'File', 'Cache'),
```

Comment:
Maybe `Type` or similar? `Family` looks strange to me here, any specific reasons for such naming?